app/scripts/Tiled1DPixiTrack.js

```diff
@@ -243,6 +243,7 @@ class Tiled1DPixiTrack extends TiledPixiTrack {
 
     const minimumsPerTile = visibleAndFetchedIds
       .map((x) => this.fetchedTiles[x])
+      .filter((tile) => tile.tileData.dense)
```

Comment:
Huh, this seems to be unrelated to the `NaN` support. What's this doing?

Reply:
Oh, good question! The test code uses the local tile fetcher which returns tile data synchronously. Somewhat paradoxically, this causes the track to try to render itself before the tile is completely filled in and throws an error if this check isn't there.